$NetBSD: patch-Makefile,v 1.3 2026/08/18 10:07:28 adam Exp $

* Allow CC override.
* Let pkgsrc define CC, optimisation, and don't force endianess.
* Honor LDFLAGS. If is provided by pkgsrc framework.

--- Makefile.orig	2019-12-17 16:05:14.000000000 +0000
+++ Makefile
@@ -4,7 +4,7 @@
 VERSION=3.3
 
 # To test strict C ANSI compliance
-CC = gcc -ansi -pedantic
+CC ?= gcc -ansi -pedantic
 LIB= -lm
 
 # This allow you to write commands like "make PURE=purify demo1"
@@ -26,14 +26,14 @@ CCPURE = $(PURE) $(CC)
 
 # If endianess of your machine is not automatically detected in Misc/common.h
 # you should manually specify here
-CFLAGS += -DLITTLE_ENDIAN
+#CFLAGS += -DLITTLE_ENDIAN
 #CFLAGS += -DBIG_ENDIAN
 
 #######################
 # GENERAL FLAGS FOR GCC
 
 # Optimized code
-CFLAGS += -O6
+#CFLAGS += -O6
 
 # Debug mode with gdb
 #CFLAGS += -g 
@@ -128,7 +128,7 @@ BINOBJS = $(BINSRCS:%.c=Bin/Standalone/%
 PROJ = mbrola
 
 $(PROJ): install_dir  $(BINOBJS) 
-	$(CCPURE) $(CFLAGS) -o $(MBRDIR)/$(PROJ) $(BINOBJS) $(LIB)	
+	$(CCPURE) $(CFLAGS) $(LDFLAGS) -o $(MBRDIR)/$(PROJ) $(BINOBJS) $(LIB)	
 
 clean:
 	\rm -f $(MBRDIR)/$(PROJ) $(PROJ).a core demo* TAGS $(BIN)/lib*.o $(BINOBJS) 
